src/client/ua_client_connect.c

```diff
@@ -11,6 +11,8 @@
 #include <open62541/transport_generated_handling.h>
 #include <open62541/types_generated_encoding_binary.h>
```

Comment:
Das ist sehr clever, was hier passiert.
Hierzu sollten wir telefonieren.

Idealerweise kann man die "Logik" knackig in einem Kommentar zu Beginn dieser Datei beschreiben.